northstar/src/runtime/state.rs

```diff
@@ -430,13 +430,15 @@ impl State {
             let events_tx = self.events_tx.clone();
             let stop = stop.clone();
             let container = Some(container.clone());
-            let permissions = manifest.console.clone();
             let connection = Console::connection(
                 runtime,
                 peer,
                 stop,
                 container,
-                permissions,
+                super::config::ConsoleQoS {
+                    permissions: manifest.console.clone(),
+                    ..Default::default()
```

Comment:
Connections from containers are never limited? Doesn't make sense because this are the only connections in a production environment. I'd have assumed here is the config from the manifest passed.

Reply:
Sorry, I was thinking the runtime was the only receiver of console connections and saw no need to limit the container side. Then remembered that there is some container to container communication :sweat_smile: